src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -150,15 +150,12 @@ private T unlink(Node<T> object) {
             head = next;
         } else {
             prev.next = next;
-            object.previous = null;
         }
         if (next == null) {
             tail = prev;
         } else {
             next.previous = prev;
-            object.next = null;
         }
-        object.item = null;
         size--;
```

Comment:
since variable "object" is unchanged in this method, we return the same  value that we took as an input
I think that return in this method is redundant, it cann as well be void.